index.bs

```diff
@@ -928,6 +930,11 @@ To <dfn noexport>parse trigger-registration JSON</dfn> given a [=string=]
     1. Set |filters| to the result of running [=parse filter data=] with
         |value|["`filters`"] and "`trigger`".
     1. If |filters| is null, return null.
+1. Let |negatedFilters| be a new [=filter map=].
+1. If |value|["`not_filters`"] exists:
+    1. Set |negatedFilters| to the result of running [=parse filter data=] with
```

Comment:
Minor nit: We could consider using non-optiona named parameters for parse filter data. It's not clear when reading the spec what `trigger` is. 

https://infra.spec.whatwg.org/#algorithm-params

"Optional named parameters, instead of positional ones, can be used to increase clarity and flexibility at the call site. Such parameters are marked up as both variables and definitions, and linked to from their call sites."

Reply:
Ack. Will do that in a followup for consistency with the other parsing.